wiki/en/en-Tips-Tricks-More.md

```diff
@@ -53,11 +53,20 @@ function loadserverstat ( $statfilename )
 
 You can save and restore the mix you have for your band rehearsals (fader, mute, pan, solo etc.) and load these any time (even while you are playing). Do this with "File > Save Mixer Channels Setup" in your client and load them using "Load Mixer Channels Setup" (or drag/drop them to the mixer window).
 
+## Delegate mixing to a mix master using the --mastermix server mode
+
+By starting a server instance with the `--mastermix` flag, the first client to connect will become the mix master.
```

Comment:
Suggest you say ", the first client to connect will become the controller, or 'master' of the audio mix." Otherwise you'll be in danger of defining a term using that term (ie "a mix master is a mix master"). Fine to refer to "mix master" after that though.